SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.spec

```diff
@@ -1,19 +1,83 @@
 %global debug_package %{nil}
 Summary:        First stage UEFI bootloader
 Name:           shim-unsigned-aarch64
-Version:        15.4
-Release:        1%{?dist}
+Version:        15
+Release:        5%{?dist}
+URL:            https://github.com/rhboot/shim
 License:        BSD
-Vendor:         Microsoft Corporation
+Vendor:         Microsoft
 Distribution:   Mariner
-URL:            https://github.com/rhboot/shim
-Source0:        https://github.com/rhboot/shim/releases/download/%{version}/shim-%{version}.tar.bz2
-Source1:        sbat.csv.in
-Source100:      cbl-mariner-ca-20210127.der
-BuildRequires:  dos2unix
-BuildRequires:  vim-extra
+Source0:        https://github.com/rhboot/shim/releases/download/15/shim-%{version}.tar.bz2
```

Comment:
Nit: I think the `Source0` tag from version 15.4 was actually more "future proof", since it used the `%{version}` macro in `download/%{version}` instead of a hard-coded `download/15`.